github/resource_github_repository.go

```diff
@@ -619,6 +619,11 @@ func resourceGithubRepositoryCreate(d *schema.ResourceData, meta interface{}) er
 		}
 	}
 
+	err := updateVulnerabilityAlerts(d, client, ctx, owner, repoName)
```

Comment:
This seems wrong, the `return resourceGithubRepositoryRead(d, meta)` line below can end up calling `updateVulnerabilityAlerts` again. Is this change actually needed?

Reply:
This part of the TF provider is a bit of a mess, AFAIK 3 months later this is required.